roles/setup_repo/tasks/validate_setup_repo.yml

```diff
@@ -15,7 +15,7 @@
   register: apt_sources
   changed_when: apt_sources.rc == '0'
 
-- name: Check for EDB Repo on RedHat
+- name: Check for EDB Repo on RedHat 1.0
```

Comment:
I find "1.0" and "2.0" confusing.  I think saying "EnterpriseDB" and "enterprisedb-enterprise" is better.

Or is it "EDB v1 repo" and "EDB v2 repo"?

Reply:
It's EDB v1 repo and EDB v2 repo. Both has different entries for EnterpriseDB Repository.

Reply:
Ok, then it should probably be EDB Repo 1.0 ...  Because it looks like RedHat 1.0. :)

Reply:
FIxed